src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes"),
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Paused").withText("shirts")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+
+        assertThat(kenshooKeywords, hasSize(2));
+    }
+
+    @Test
+    public void testDownloadActiveGoogleKeyword(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes")
```

Comment:
For this tested scenario, the text and the id are not meaningful and only makes the test bigger.
Maybe we could remove them:
```
setupKeywordsInGoogle(keyword().withCampaignId(1000).withStatus("Active"));;
```
In case we can't, (e.g. when implementation can't handle missing ID), then the method keyword() could construct a builder with some defaults.
This will enable us writing a test scenario that concentrates only in the status.